src/components/Typography/BodyX/BodyX.module.css

```diff
@@ -16,6 +16,10 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+.body {
```

Comment:
Can we apply this style using Ant design tokens?

Reply:
For paragraphs, there are no ad hoc display tokens. However, having used those for titles, here I put `margin: 0 !important` inside the individual `bodyS`, `bodyM`, `bodyL` so as not to create more classes than necessary